osu.Game/Extensions/DrawableExtensions.cs

```diff
@@ -79,5 +80,40 @@ public static void ApplySkinnableInfo(this Drawable component, SkinnableInfo inf
                     container.Add(child.CreateInstance());
             }
         }
+
+        /// <summary>
+        /// Keeps the drawable upright and unstretched preventing it from being rotated, sheared, scaled or flipped with its Parent.
+        /// </summary>
+        /// <param name="drawable">The drawable.</param>
+        public static void KeepUprightAndUnscaled(this Drawable drawable)
```

Comment:
Given that `KeepUprightAndUnscaled()` is basically useless without a `GrowToFitContainer`, I see no point in making it a publically-accessible method. It should be inside `UprightAndUnscaledContainer` as private.